agent/yubiagent/generate_test.go

```diff
@@ -4,6 +4,7 @@
 package yubiagent
 
 //go:generate mockgen -destination=./mock_shimagent_test.go -package=yubiagent github.com/theparanoids/ysshra/agent/shimagent ShimAgent
+//go:generate mockgen -destination=./mock_yubiagent.go -package=yubiagent github.com/theparanoids/ysshra/agent/yubiagent YubiAgent
```

Comment:
instead of `mock_yubiagent_test.go` because we want to reuse it in other packages.

Reply:
As the is a mock, maybe put it in a mock package.